test/tls/tests_tls_netaccess.uts

```diff
@@ -83,26 +88,30 @@ test_tls_server("ECDHE-RSA-AES256-GCM-SHA384", "-tls1_2")
 
 = Load client utils functions
 
-import sys, os, threading, Queue
+import sys, os, threading
+try:
+    from Queue import Queue
+except ImportError:
```

Comment:
You may use scapy.modules.six.moves.queue

Reply:
Done